presto-plugin-toolkit/src/main/java/io/prestosql/plugin/base/authentication/KerberosAuthentication.java

```diff
@@ -109,4 +111,13 @@ private static Configuration createConfiguration(String principal, String keytab
             }
         };
     }
+
+    private static String getServerPrincipal(String principal, String hostname)
```

Comment:
this is a copy of org.apache.hadoop.security.SecurityUtil#getServerPrincipal(java.lang.String, java.lang.String)